pymatgen/analysis/phase_diagram.py

```diff
@@ -245,7 +280,7 @@ def as_dict(self):
             "@module": self.__class__.__module__,
             "@class": self.__class__.__name__,
             "entry": self.original_entry.as_dict(),
-            "composition": self.composition,
+            "sp_mapping": self.sp_mapping,
```

Comment:
this may be a breaking change for workflows storing transformed entries as dictionaries

Reply:
@shyamd do you have any comments on this before merged?

Reply:
TransformedPDEntries are usually only used when doing alternative phase diagrams, so this shouldn't affect me.